stories/theme/ag/small/ListingCell.css

```diff
@@ -0,0 +1,118 @@
+.ListingCell-map {
+  width: 280px;
+  height: 200px;
+  border-radius: 5px;
+  background-color: #ffffff;
+  box-shadow: 0 4px 4px 0 rgba(0, 0, 0, 0.12), 0 0 4px 0 rgba(0, 0, 0, 0.24);
+  font-family: ProximaNova;
+  color: #9b9b9b;
+}
+
+.ListingCell-map .Card_Body {
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-end;
+}
+
+.ListingCell_Top {
+  height: 120px;
+  border-bottom: 1px solid lightgrey;
+}
+
+.ListingCell_Bottom {
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-end;
+  height: 80px;
+  padding: 5px 6px 11px 10px;
+}
+
+.ListingCell_Carousel {
+  height: 100%;
+  width: 100%;
+  text-align: center;
+}
+
+.ListingCell_Coupon {
+  height: 16px;
+  width: 54px;
+  position: relative;
+  bottom: 120px;
+}
+
+.ListingCell_Favorite {
+  height: 16px;
+  width: 15px;
+  position: relative;
+  bottom: 135px;
+  left: 218px;
+}
+
+.ListingCell_Info_Bottom,
```

Comment:
You should never have 2 `_`s in a classname.  If you do, either the syntax is wrong, or your component can be broken down further.